### DISCUSSION TITLE
Is ABB Terra AC 22kw supported?

### DISCUSSION THREAD

**Discussion by xaviondk** [lbbrhzn/ocpp#654] [Q&A]
Hey.

Im looking at buying my first wall charger, and have a plan to dive into HomeAssistant when the HA Yellow PoE version is ready to ship.

I see that the 7 and 11kw Terra AC's are on the supported list, so can I assume that the OCPP communication will also be working for the 22kw version TAC-W22-T-RD-MC-0?

Thanks

**Comment by RPJacobs** [comment]
Yes configured it today. Works like a charm.

**Reply by BDSmmadsen** [reply]
Hello, 
Have a few questions.

Do the ABB app work aswell when using HA OCPP ?
My W22 keeps blinking in Internet light on the charger when using my HA adress.

Do you use http:// or ws:// for OCCP on the charger?

**Reply by RPJacobs** [reply]
Yes it works. I used http://

**Reply by BDSmmadsen** [reply]
You dont get the constant reboot and crash messages as notifications in HA ?

**Reply by RPJacobs** [reply]
Nope it works like a charm

**Reply by BDSmmadsen** [reply]
Do you have a steady "Internet" light on the charger aswell?
And did you get it configured via the old TerraConfig app? Where you had to contact ABB to make them add the OCPP server

**Reply by RPJacobs** [reply]
I used the terra config (legacy) app. Did not contact ABB, changed the server bij doing a MITM with Charles proxy.

**Reply by anticam** [reply]
With the new Terra Config app I could request an account and get access. In Terra Config, the HA OCPP server could be set.

**Reply by Finki22** [reply]
How do i have to configure the Ocpp server in the Terra AC
Where do i get the password from? And a certificate

**Reply by RPJacobs** [reply]
Use the app.

**Reply by Finki22** [reply]
The TerraConfig app? or the ChargerSync app?

**Reply by anticam** [reply]
The TerraConfig app, the new one not the legacy, when it starts it offers a registration link, then I received a confirmation email to log in. I did not configure certificate, plain HTTP is used currently.

**Reply by Finki22** [reply]
Thanks for the hint!
I tried with the app but i get timeouts when i want to write my configuration.
Maybe i have to wait for the next update.

can you tell me how to specify the url? ip:port ?

**Reply by anticam** [reply]
With the ChargerSync the ABB Terra AC was updated to the latest firmware. 
In HACS OCCP integration, the port number  was set, default is 9000.
https://home-assistant-ocpp.readthedocs.io/en/latest/installation.html#install-the-ocpp-repository

In TerraConfig I set local OCCP server section
http://192.158.1.166:9000
the port number is the same configured in the OCCP integration.
https://guides.e-mobility.abb.com/TerraConfig2MobileApp_EN/#/lessons/nG1Xg9-fLCZAwLd1cp2ozme16o34myNH

**Comment by LarsBorris** [comment]
I also installed this one and looks like its talking with the OCPP server.

**Reply by BDSmmadsen** [reply]
Hello,
Have a few questions.

Do the ABB app work aswell when using HA OCPP ?
My W22 keeps blinking in Internet light on the charger when using my HA adress.

Do you use http:// or ws:// for OCCP on the charger?
